R/independence_corr_func.R

```diff
@@ -0,0 +1,24 @@
+#' Generates the correlation matrix for a non-spatial process.
+#'
+#' @param corr_function_params NULL
+#'
+#' @return Correlation matrix of diagonal ones.
+#'
+independence_corr_func <- function(
```

Comment:
I see now, so essentially you made this function to handle the case when we don't want spatial correlation. I think we can specify this a little more simply by adding a switch, curious your thoughts @cbernalz @dylanhmorris